Remotion/ObjectBinding/Web/UI/Controls/BocReferenceValueImplementation/Rendering/BocReferenceValueRendererBase.cs

```diff
@@ -509,7 +509,7 @@ protected string GetValidationErrorsID (BocRenderingContext<TControl> renderingC
 
     private string GetCssClassInnerContent (BocRenderingContext<TControl> renderingContext)
     {
-      string cssClass = CssClassInnerContent;
+      string cssClass = CssClassInnerContent + " " + CssClassDefinition.RemotionThemed;
```

Comment:
If we define the property on the base renderer (see BocRendererBase), we can use the property here.